docker-compose.yaml

```diff
@@ -7,6 +7,7 @@ services:
       - TRUID_CLIENT_ID
       - TRUID_CLIENT_SECRET
       - TRUID_BASEPATH
+      - TRUID_REDIRECT_URI
```

Comment:
Do you need to set TRUID_REDIRECT_URI, isn't it possible to get from EXAMPLE_DOMAIN?

Reply:
They are different for app 2 app in ios. There i need http://johannes-computer.local as EXAMPLE_DOMAIN and truidtest://truid/v1... as redirect_uri